shells/pipes-shell/BUILD

```diff
@@ -6,12 +6,24 @@ run_in_repo(
     name = "pipes_shell_web",
     srcs = glob(["**"]),
     outs = ["pipes_shell_web_dist"],
-    cmd = "cd shells/pipes-shell/web/deploy && " +
+    cmd = "set OLDPWD=$PWD && cd $PWD/$(dirname $(location //shells/pipes-shell:web_deploy_bin)) && " +
           "./deploy.sh && " +
-          "cp -r dist {OUT}",
+          "cp -r dist $OLDPWD/{OUT}",
     progress_message = "Webpacking pipes-shell",
     visibility = ["//visibility:public"],
     deps = [
-        "//tools:sigh_webpack",
+        "//tools:sigh_bin", 
```

Comment:
I don't think this is right. The old version used to run sigh webpack, before running deploy.sh. The Arcs runtime needs to be built before you can deploy the shell. If you change runtime code and then run this build rule again, it won't rebuild anything.

I think you need to revert this to depend on sigh_webpack